includes/wc-functions.php

```diff
@@ -1265,3 +1265,25 @@ function dokan_modify_vendor_order_counts( $counts ) {
 }
 
 add_filter( 'wp_count_posts', 'dokan_modify_vendor_order_counts', 10, 1 );
+
+
+/**
+ * PDF Invoices & Packing Slips for WooCommerce plugin integration on suborder section.
+ *
+ * @since DOKAN_SINCE
+ *
+ * @param $actions
+ * @param $order
+ *
+ * @return mixed
+ */
+function dokan_pdf_invoice_button( $actions, $order ) {
```

Comment:
@aihimel Can you please move this function under includes/Order/Frontend/Hooks.php file?

Reply:
moved